### PR TITLE
Update few dev related docs and reframe titles in nav bar

### PR DESCRIPTION
**Pull request by logason**

### CODE REVIEW

File: components/Navigation.tsx [hunk 1]
```diff
@@ -214,72 +214,63 @@ const navigation: navigationItem[] = [
       },
       {
         type: 'route',
-        title: 'Reading the implementation diff',
-        path: '/implementation/read-implementation-diff',
+        title: 'Using the Inspector SDK',
+        path: '/implementation/setup-inspector-sdk',
```
Comment (hugihlynsson) [inline components/Navigation.tsx:218]: Do we want to add redirects for these changed urls?

File: pages/implementation/avo-and-git.mdx [hunk 2]
```diff
@@ -1,37 +1,25 @@
...
+## Workflow example
 
 Let’s look at an example. In our fictional company we have two development teams, iOS and Android. Even though the iOS and Android teams have an almost identical longterm product roadmap, they don’t work on the same features at the same time. In addition to that, each platform team can also be working on multiple features at the same time. This means there are always multiple analytics tracking changes happening in the company simultaneously.
 
 How does that work in Avo, where there is a single tracking plan shared across the entire company, and directly connected to each codebase with code generated Avo Functions to ensure a single source of truth?
 
-### Branches in Avo 
+### Avo branches 
+
+Developers have been using branches in git to work on multiple code changes simultaneously for almost 20 years. In Avo you can use branches too, to work on multiple tracking plan changes simultaneously, both when designing your tracking plan and when implementing the tracking in code.
```
Comment (hugihlynsson) [inline pages/implementation/avo-and-git.mdx:18]:
```suggestion
Developers have been using branches in Git to work on multiple code changes simultaneously for almost 20 years. In Avo you can use branches too, to work on multiple tracking plan changes simultaneously, both when designing your tracking plan and when implementing the tracking in code.
```

File: pages/implementation/avo-and-git.mdx [hunk 3]
```diff
@@ -52,20 +40,49 @@ _Onboarding flow event in Avo, sent from iOS_
...
 
-Once the tracking has been implemented, and the feature is ready to go, the developer gets their git branch merged like they usually would. As soon as the git branch is merged into the git main branch, they merge their Avo branch in the Avo main branch. By doing so, the Avo main branch always represents the current state of Avo tracking on the git main branch on every platform.
+When it’s time for the Android team to implement the same changes, they would go through the same flow, but instead of creating the events and properties in Avo, they would only need to add the Android source to the events they want to implement, set the properties to be always sent from Android and run `avo pull` from their Avo branch to implement the tracking changes.
```
Comment (hugihlynsson) [inline pages/implementation/avo-and-git.mdx:53]: Do they need to explicitly set the properties to be always sent on Android, won't that happen automatically?

File: pages/implementation/avo-and-git.mdx [hunk 4]
```diff
@@ -52,20 +40,49 @@ _Onboarding flow event in Avo, sent from iOS_
...
 
-When it’s time for the Android team to implement the same changes, they would go through the same flow, but instead of creating the events and properties in Avo, they would only need to add the Android source to the events they want to implement, and set the properties to be always sent from Android.
+Previously merged Avo branches can always be viewed back in time. That means you can always see what changes were made when, and by whom.
```
Comment (hugihlynsson) [inline pages/implementation/avo-and-git.mdx:55]:
```suggestion
Previously merged Avo branches can always be viewed back in time. That means you can always see the state of your tracking plan at that point, what changes were made when, and by whom.
```

**Review by hugihlynsson — APPROVED** [review]
Few small suggestions, otherwise impeccable 👌

File: pages/implementation/avo-and-git.mdx (hunk 3 above)
Comment (logason) [reply pages/implementation/avo-and-git.mdx:53]: We could potentially skip this detail here, but earlier in the guide they mark the property as sent from iOS and not Android

**Pull request merged**